### DISCUSSION TITLE
2024/12/29/20241229_updatelog/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#33] [General]
# 2024/12/29/20241229_updatelog/

又是一年岁末。似乎又到了写一年一度年终总结的季节。 然而相比于往年，今年的岁末有些过于平静甚至平庸。最近这几周，在忙碌于科研工作与助教课和摆烂于互联网两种状态之间来回切换，实在记不清有什么值得记录的事情。 好在，周末回看了一整年的博客和APP年终报告，突然觉得这一年经历了好多事情。所以这篇博客，就当是一篇年终盘点吧。

https://wz.anoms.top/2024/12/29/20241229_updatelog/

**Comment by THUDBT** [comment]
已阅

**Comment by THUDBT** [comment]
个站跟影响力从来无缘，还不如在pyq发。建议不弄自媒体，内容太专业，没人看的。想有人看就得费大力气钻研流量、内容垂直、粉丝受众等，还需要有持续输出干活的能力，也许跟你初心不符。

**Reply by THUDBT** [reply]
* 干货

**Reply by cyclinbox** [reply]
学习到了，谢谢樊总！
